$NetBSD$

Add /usr/pkg/share/applications to paths searched for .desktop files

--- i3-dmenu-desktop.orig	2016-03-24 17:50:36.000000000 +0000
+++ i3-dmenu-desktop
@@ -111,7 +111,7 @@ $xdg_data_home = $ENV{HOME} . '/.local/s
     ! -d $xdg_data_home;
 
 my $xdg_data_dirs = $ENV{XDG_DATA_DIRS};
-$xdg_data_dirs = '/usr/local/share/:/usr/share/' if
+$xdg_data_dirs = '/usr/local/share/:/usr/share/:/usr/pkg/share/' if
     !defined($xdg_data_dirs) ||
     $xdg_data_dirs eq '';
 
